examples/aspnet/OpenApiJsonDocumentFilter.cs

```diff
@@ -59,10 +48,24 @@ public void Apply(OpenApiDocument swaggerDoc, DocumentFilterContext context)
                     Summary = "OAS",
                     Responses = new OpenApiResponses
                     {
-                        ["200"] = response
+                        ["200"] = openApiResponse
                     }
                 }
             }
         });
+
+        foreach (var pathResponse in document.Paths.Values.SelectMany(path => path.Operations.Values.SelectMany(operation => operation.Responses)))
```

Comment:
We loopen hiermee dynamisch door alle paden en voegen de header toe. Daardoor hoeven we die niet meer expliciet toe te voegen aan de `openapiHeaders`